### PR TITLE
Switch to Google Tag Manager

### PR DESCRIPTION
**Pull request by jacksoncvm**

### CODE REVIEW

**Comment by jacksoncvm** [comment]
This is in preparation of moving you to your own ZeroTracker instance of TrustArc

File: site/js/ga-bootstrap.js [hunk 1]
```diff
@@ -1,37 +1,37 @@
...
 
- var _gaq = _gaq || [];
-  _gaq.push(['_setAccount', 'UA-1001800-1']);
-  _gaq.push(['_trackPageview']);
-  (function() {
-  var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
-  ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
-  var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
-  })();
+//  var _gaq = _gaq || [];
```
Comment (lukebakken) [inline site/js/ga-bootstrap.js]: Can this code be deleted rather than commented out?

Comment (jacksoncvm) [reply site/js/ga-bootstrap.js]: Sure - we can delete that file altogether

**Comment by jacksoncvm** [comment]
Yes, we can delete that file.

-Chloe

**Comment by lukebakken** [comment]
I've deleted that file and opened #554 with these changes.

**Comment by lukebakken** [comment]
@jacksoncvm merged in #554, thanks!

**Pull request closed without merging**